src/index.html

```diff
@@ -57,5 +58,8 @@ <h1 class="my-container">MediaHub, A place to like and comment on your favourite
     <section id="pop-up" class="my-container">
        <!-- Popup will go here -->
       </section>
+      <script type="text/javascript">
+        // document.getElementById('num').innerHTML = movieList.length;
+      </script>
```

Comment:
Please remove the script tag. It is best practice for all Javascript code to be written in a separate JS file.